### PR TITLE
Updates to Mythological Weapons Pack replicas, recipe formatting

### PR DESCRIPTION
**Pull request by DangerNoodle**
Closes https://github.com/CleverRaven/Cataclysm-DDA/issues/18323

Updates the Ichaival replica to use copy-from to stay current with changes to base item, a reflex recurve bow. Additionally updates all weapons in the mod to use the same setup.

Also changed the Laevateinn replica to use the arming sword as a base, with stat and recipes changes to reflect this.

Finally, recipes should be in line with current style.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
When I was bugfixing Mythological Weapons I found it lacked mod slots as well.

**Comment by DangerNoodle** [comment]
Ah right. Should I go ahead and made this a general update to base items as well?

**Comment by BorkBorkGoesTheCode** [comment]
The STR_12 flag can be replaced by "STR_DRAW", flag and "min_strength":. You might add PRIMITIVE_RANGED_WEAPON as well, to allow repairing without gunsmithing tools

**Comment by DangerNoodle** [comment]
Ah right.

**Comment by BorkBorkGoesTheCode** [comment]
I wonder: could Laevateinn be turned into a manhack which can be used as a tool when deactivated?

**Comment by DangerNoodle** [comment]
That would be rather odd. As far as I can tell, the weapons in this mod pack only gain mythical traits by way of having slightly better stats than the base item. And in the case of damage, it is presumably to rebalance damage per turn relative to the increased weight from decoration.

That might be a better fit for the Arcana and Magic Items mod.

**Comment by mugling** [comment]
> Simply gives the Ichaival replica the same armor data as its base item, a reflex recurve bow.

Consider `copy-from` syntax

**Comment by DangerNoodle** [comment]
That might be a better idea, yes. Sorry for the delay, will work on that in a moment.

File: data/mods/NW_Pack/nw_archery.json [hunk 1]
```diff
@@ -1,27 +1,17 @@
...
-      "sight_dispersion": 150,
-      "recoil" : 0,
-      "durability" : 6,
-      "clip_size" : 1,
-      "reload" : 80,
-      "valid_mod_locations": [["underbarrel", 1]]
-   }
+  {
+    "id" : "ichaival_replica",
+    "copy-from": "gun_base",
```
Comment (mugling) [inline data/mods/NW_Pack/nw_archery.json]: Shouldn't that instead be from reflex bow?

Comment (DangerNoodle) [inline data/mods/NW_Pack/nw_archery.json]: Copy-paste failure on my part, apologies.

**Pull request merged**